docs/UserGuide.md

```diff
@@ -512,9 +519,9 @@ Format: `close Option`
 
 Examples:
 * `close -c`
-    * close contact panel
+    * The contact panel will be invisible to the user
```

Comment:
very minor, but contact should be capitalized for consistency (as in line 500)

Reply:
resolved